### ISSUE TITLE
Inconsistent spacing with several albums of varying track count

### ISSUE THREAD

**Issue by MineyMe** [gnumdk/lollypop#48]
Hello

I noticed that the spacing between the track length and its title varies depending on the amount of columns (i.e. the first album in the screenshot has two columns and smaller spacing whilst the second album has one column and larger spacing). I shrank the window before creating this screenshot, but the difference is even more noticable on a larger window width. I do not know if this is intended behavior, but it looks a little out of place if you have alternating albums of varying track counts :P.

![spacing](https://cloud.githubusercontent.com/assets/6823929/6651513/8cd198a8-ca46-11e4-9acf-86beefb5ca79.png)

Thanks in advance

**Comment by gnumdk** [comment]
I know this bug but for albums with one track... What is happening here, strange :-(

**Comment by gnumdk** [comment]
Can you send me your ~/.local/share/lollypop/lollypop.db by email et tell me what album is having this bug.

Thanks.

**Comment by MineyMe** [comment]
Seeing as I had strange tags last time, I went to check them again. It turns out there is something strange about the tracks affected by this bug: The album you see on top is the original album, whilst the second album are actually four tracks from the first album with a few audio operations applied. Naturally I gave them the same tags when I modified them and Lollypop does (neatly) place them as separate albums, even though the tags are the same. The only tag that is different is the 'Genre' tag. I noticed also that if I retag two of the tracks to use a different album name (so they are no longer the same), then Lollypop notices three albums in total:
  * The original album (correctly formatted).
  * The two tracks that now have a different album (now correctly formatted!)
  * The remaining tracks that still have the same album (still incorrectly formatted).

So it again appears my shady tagging is the culprit, my apologies. I'm however not entirely sure if what currently happens is the expected behavior, but I'm also not sure what I would expect to happen instead.
